index.html

```diff
@@ -56,6 +56,127 @@ <h1>
+            <ul class="work-languages" >
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+            </ul>
+            <button type="button" > See Project</button>
+        </div> 
+    </div>
+    <div class="myworks-card">
+        <div class="myworks-thumbnail">
+            <img  alt="">
+            <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+            <ul class="work-languages" >
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+            </ul>
+            <button type="button" > See Project</button>
+        </div> 
+    </div>
+    <div class="myworks-card">
+        <div class="myworks-thumbnail">
+            <img  alt="">
+            <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+            <ul class="work-languages" >
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+            </ul>
+            <button type="button" > See Project</button>
+        </div> 
+    </div>
+</section>
+<section class="about-me">
+    <div class="container-aboutme">
+        <h2>About me</h2>
+        <div class="aboutme-details">
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+			see and have a project you need coded,
+			don’t hestiate to contact me.</p>
+        </div>
+        <button type="button" >Get My Resume</button>
+    </div>
+    <div class="attributes-and-skills">
+        <ul class="my-attributes">
+            <li>
+                <div class="attribute">
+                    <div class="attribute-image">
+                        <img src="./images/icons/languages-icon.svg" alt="Languages icon">
+                    </div>
+                    <h2 class="attribute-name">Languages</h2>
+					<ul class="attribute-list">
+						<li>JavaScript</li>
+						<li>Ruby</li>
+						<li>Html</li>
+						<li>CSS</li>
+					</ul>
+                </div>
+            </li>
+            <li>
+                <div class="attribute">
+                    <div class="attribute-image">
+                        <img src="./images/icons/frameworks-icon.svg" alt="Frameworks icon">
+                    </div>
+                    <h2 class="attribute-name">Frameworks</h2>
+					<ul class="attribute-list">
+						<li>React.js</li>
+						<li>Ruby on Rails</li>
+						<li>RSpec</li>
+						<li>Capybara</li>
+                        <li>Selenium</li>
+					</ul>
+                </div>
+            </li>
+            <li>
+                <div class="attribute">
+                    <div class="attribute-image">
+                        <img src="./images/icons/skills-icon.svg" alt="Skills icon">
+                    </div>
+                    <h2 class="attribute-name">Skills</h2>
```

Comment:
- [ ] Great job on your skills section! I noticed that the alignment of two of your skill cards is a bit off compared to the Figma template design. It would be great if you could adjust the alignment of those images to match the design. 🙋‍♂️

### Your design:
![image](https://user-images.githubusercontent.com/107952724/235262995-f02c7bf9-aca7-4863-abc3-a3f4ef38194c.png)

### Figma design:
![image](https://user-images.githubusercontent.com/107952724/235263123-c97885c8-f012-4201-97f9-c23f241709b0.png)

Reply:
@taldr27 please elaborate the alignment issue, i couldn't see the difference

Reply:
Of course @demesameneshoa, happy to help! To elaborate on the alignment issue you mentioned, would you mind checking the distance between the border of your cards and the circle image? I'll leave you a couple of images below. Let me know if you need any further assistance! 💪

### Your cards:
![image](https://user-images.githubusercontent.com/107952724/235267556-53c89481-cde7-4083-a7b6-6679362c58d3.png)

### Figma cards:
![image](https://user-images.githubusercontent.com/107952724/235267639-59321b1a-2fcd-4562-8567-d65f542ccb01.png)